### PR TITLE
add role and rolebinding defs

### PR DESCRIPTION
**Pull request by j18e**

### CODE REVIEW

**Review by birgirst — DISMISSED** [review]

File: k8s/models/role.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+from ..base import Model
+from ..fields import Field, ListField
+
+
+class PolicyRule(Model):
+    api_groups = ListField(str)
+    resources = ListField(str)
+    verbs = ListField(str)
+    resourceNames = ListField(str)
+    nonResourceUrls = ListField(str)
```
Comment (oyvindio) [inline k8s/models/role.py:29]: Should this be `nonResourceURLs`?

File: k8s/models/role_binding.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+from .common import ObjectMeta
+from ..base import Model
+from ..fields import Field, ListField
+
+
+class RoleRef(Model):
+    apiGroup = Field(str)
+    kind = Field(str)
+    roleRef = Field(str)
```
Comment (oyvindio) [inline k8s/models/role_binding.py:27]: As far as I can see, RoleRef doesn't have a `roleRef` field, but it has a `name` field which should probably be added here

File: k8s/models/role_binding.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+class Subject(Model):
+    kind = Field(str)
+    name = Field(str)
+    apiGroup = Field(str)
+    namespace = Field(str)
```
Comment (oyvindio) [inline k8s/models/role_binding.py:34]: As above; string fields should use six.text_type

**Review by oyvindio — CHANGES_REQUESTED** [review]
I've left some comments inline. I think it would also be really nice to have some simple get/update/delete etc. tests for these models, similar to those that exist for other model classes here https://github.com/fiaas/k8s/tree/master/tests/k8s

**Review by j18e — COMMENTED** [review]
these have been done

**Review by oyvindio — APPROVED** [review]
Thanks for adding tests! This looks good 👍

**Pull request merged**